asgi_ipc.py

```diff
@@ -42,15 +42,11 @@ def __init__(self, prefix="asgi", expiry=60, group_expiry=86400,
         )
         self.thread_lock = threading.Lock()
         self.prefix = prefix
-        self.channel_store = MemoryDict(
-            "/%s-chan" % self.prefix,
-            size=channel_memory
-        )
+        path = Path('/tmp') / Path('{prefix}.sqlite'.format(prefix=prefix))
+        connection = sqlite3.connect(str(path))
+        self.message_store = MessageTable(connection, '{}-message'.format(prefix))
```

Comment:
I used this to build the semaphore name/path, but I think I'll just switch to `'/{table-name}-sem'`.